src/api-service/__app__/onefuzzlib/notifications/ado.py

```diff
@@ -209,15 +209,23 @@ def create_new(self) -> None:
                 self.project,
                 entry.id,
             )
+        return entry
 
-    def process(self) -> None:
+    def process(self, notification_info: str) -> None:
         seen = False
         for work_item in self.existing_work_items():
             self.update_existing(work_item)
```

Comment:
Is there a way to identify if the update fails?

Reply:
an exception will be thrown if it fails